### PR TITLE
Added skiller_sgk50_s4 keyboard

### PR DESCRIPTION
**Pull request by itarze**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sharkoon/skiller_sgk50_s4/info.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false
```
Comment (waffle87) [inline keyboards/sharkoon/skiller_sgk50_s4/info.json:22]:
```suggestion
        "rgb_matrix": true
```

File: keyboards/sharkoon/skiller_sgk50_s4/info.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        "device_version": "1.0.0",
+        "pid": "0x1020",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x6332"
+    },
+    "ws2812": {
+        "pin": "A8"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (waffle87) [inline keyboards/sharkoon/skiller_sgk50_s4/info.json:158]:
```suggestion
        "LAYOUT": {
```

File: keyboards/sharkoon/skiller_sgk50_s4/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
+    ),
+
+    [1] = LAYOUT_all(
```
Comment (waffle87) [inline keyboards/sharkoon/skiller_sgk50_s4/keymaps/default/keymap.c:15]:
```suggestion
    [0] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
    ),

    [1] = LAYOUT(
```

File: keyboards/sharkoon/skiller_sgk50_s4/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
+    ),
+
+    [1] = LAYOUT_all(
```
Comment (waffle87) [inline keyboards/sharkoon/skiller_sgk50_s4/keymaps/via/keymap.c:15]:
```suggestion
    [0] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
    ),

    [1] = LAYOUT(
```

**Comment by dunk2k** [comment]
layout information, in `info.json`, doesn't match image in `readme.md` - are there ISO and ANSI variants of this keyboard?

**Comment by dunk2k** [comment]
@itarze 
Could you confirm matrix co-ordinates (e.g., `1,13`) for the following keys?:
- ISO Enter
- ISO Hash
- ANSI/2.75u Left Shift

**Comment by itarze** [comment]
@dunk2k 
> * ISO Enter  2,13
> * ISO Hash  2,12
> * ANSI/2.75u Left Shift 3,0

**Comment by dunk2k** [comment]
> @dunk2k
> 
> > * ISO Enter  2,13
> > * ISO Hash  2,12
> > * ANSI/2.75u Left Shift 3,0

thank you kindly, code suggestion(s) inbound

File: keyboards/sharkoon/skiller_sgk50_s4/info.json [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+        "device_version": "1.0.0",
+        "pid": "0x1020",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x6332"
+    },
+    "ws2812": {
+        "pin": "A8"
+    },
+    "layouts": {
+        "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s4/info.json:157]:
```suggestion
    "community_layouts": ["60_ansi", "60_iso"],
    "layouts": {
        "LAYOUT_all": {
```

File: keyboards/sharkoon/skiller_sgk50_s4/info.json [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 12], "x": 12.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 13], "x": 13.75, "y": 4, "w": 1.25}
+            ]
+        }
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s4/info.json:222]:
```suggestion
        },
        "LAYOUT_60_ansi": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [2, 12], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 2], "x": 2.25, "y": 3},
                {"matrix": [3, 3], "x": 3.25, "y": 3},
                {"matrix": [3, 4], "x": 4.25, "y": 3},
                {"matrix": [3, 5], "x": 5.25, "y": 3},
                {"matrix": [3, 6], "x": 6.25, "y": 3},
                {"matrix": [3, 7], "x": 7.25, "y": 3},
                {"matrix": [3, 8], "x": 8.25, "y": 3},
                {"matrix": [3, 9], "x": 9.25, "y": 3},
                {"matrix": [3, 10], "x": 10.25, "y": 3},
                {"matrix": [3, 11], "x": 11.25, "y": 3},
                {"matrix": [3, 13], "x": 12.25, "y": 3, "w": 2.75},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 12], "x": 12.5, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13.75, "y": 4, "w": 1.25}
            ]
        },
        "LAYOUT_60_iso": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 12], "x": 12.75, "y": 2},
                {"matrix": [2, 13], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [3, 1], "x": 1.25, "y": 3},
                {"matrix": [3, 2], "x": 2.25, "y": 3},
                {"matrix": [3, 3], "x": 3.25, "y": 3},
                {"matrix": [3, 4], "x": 4.25, "y": 3},
                {"matrix": [3, 5], "x": 5.25, "y": 3},
                {"matrix": [3, 6], "x": 6.25, "y": 3},
                {"matrix": [3, 7], "x": 7.25, "y": 3},
                {"matrix": [3, 8], "x": 8.25, "y": 3},
                {"matrix": [3, 9], "x": 9.25, "y": 3},
                {"matrix": [3, 10], "x": 10.25, "y": 3},
                {"matrix": [3, 11], "x": 11.25, "y": 3},
                {"matrix": [3, 13], "x": 12.25, "y": 3, "w": 2.75},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 12], "x": 12.5, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13.75, "y": 4, "w": 1.25}
            ]
        }
```

File: keyboards/sharkoon/skiller_sgk50_s4/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
+    ),
+
+    [1] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s4/keymaps/default/keymap.c:15]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
    ),

    [1] = LAYOUT_all(
```

File: keyboards/sharkoon/skiller_sgk50_s4/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
+    ),
+
+    [1] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s4/keymaps/via/keymap.c:15]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,    KC_6,    KC_7,     KC_8,     KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,    KC_H,    KC_J,     KC_K,     KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,     KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT,                             KC_SPC,                               KC_RALT, MO(1),   KC_APP,  KC_RCTL
    ),

    [1] = LAYOUT_all(
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/sharkoon/skiller_sgk50_s4/info.json (hunk 2 above)
Comment (drashna) [reply keyboards/sharkoon/skiller_sgk50_s4/info.json:158]: @waffle87 this has multiple layouts, so doesn't the "all" apply here?

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by vsmaxim** [comment]
@itarze 

Hello there! 

I've been attempting to flash the default firmware to the mentioned keyboard. However, I can't figure out how to enter the bootloader. Would it be possible to check on what's the right way to do that?

My keyboard PCB looks like [this one](https://www.techpowerup.com/review/sharkoon-skiller-sgk50-s4-mechanical-keyboard/4.html)

I've tried the following:
* **Bootmagic reset** - this doesn't work, keyboard loads as usual
* **Physical reset button** 
    * there's no physical reset button sadly
    * I've tried shorting two pads right to space switch
      * keyboard connects
      * almost no lights, only l-ctrl, l-alt and space are slightly dimming
      * USB ID `0C45:7010:0100`
      * `wb32-dfu-updater_cli.exe -l` shows `Not found device!`
* **Keycode in layout** - couldn't find any

**Comment by itarze** [comment]
@vsmaxim It seems that your keyboard is not a QMK version. Maybe it's just that the keyboard shares the same name.

**Pull request merged**